packages/ui/src/theme/overwrites/components/Checkbox.ts

```diff
@@ -0,0 +1,42 @@
+import { mode } from '@chakra-ui/theme-tools';
+
+const parts = ['container', 'control', 'label', 'icon'];
+
+function baseStyleControl(props: Record<string, any>) {
+  const { colorScheme: c } = props;
+  const disabledCbColorLight = 'gray.100';
+
+  return {
+    _checked: {
+      _active: {
+        bg: mode(`${c}.700`, `${c}.200`)(props),
```

Comment:
I understand that using mode here makes this component more adaptable. Nevertheless, we don't test this component on dark mode anywhere so we cannot be sure if it looks good. In my opinion, it would be better to just pass string values here. We would have an additional gain as there will be no need to use the function to generate styles and that could make code easier to read

Reply:
I think, as long as we have all styles in one place (theme) it wouldn't be hard to add a dark theme in the future when we will have mockups for it